models/Element/ElementInterface.php

```diff
@@ -208,11 +212,24 @@ public function getLocked();
      */
     public function getParentId();
 
+    /**
+     * @param int|null $id
+     *
+     * @return $this
+     */
+    public function setParentId($id);
+
     /**
      * @return self|null
      */
     public function getParent();
 
+    /**
+     * @param ElementInterface|null $parent
+     * @return static
```

Comment:
Shouldn't this method return `$this`, too?

```suggestion
     *
     * @return $this
```

Reply:
You were a few seconds faster :)